### ISSUE TITLE
Only Countries does not work

### ISSUE THREAD

**Issue by nezaboravi** [EducationLink/vue-tel-input#166]
Hi, 
```
<vue-tel-input
                                                :onlyCountries="['AU', 'BR']"
                                                wrapperClasses="h-12 border-b-1 border-red-500 w-96"
                                                mode="international"
                                                autocomplete="username"
                                                inputClasses="w-full"
                                                name="tel"
                                                :value="{{json_encode($user->tel)}}"
                                            />
```
It shows all countries, regardless.
This is in laravel blade view.
Tried to set default settings in app.js like so:
```

import VueTelInput from 'vue-tel-input'

Vue.use(VueTelInput, [{
    onlyCountries: ['AU', 'BR']
}])

```

It does not work too, it shows all countries.

**Comment by imjonos** [comment]
I have this issue too

**Comment by iamstevendao** [comment]
Should be fixed in [`v4.4.0`](https://github.com/EducationLink/vue-tel-input/releases/tag/v4.4.0)
